### PR TITLE
Fix/move composer

### PR DESCRIPTION
**Pull request by a-laurowski**
- Solves issue:  #15
- Description: Correct module installation by composer

### CODE REVIEW

**Comment by fritzmg** [comment]
Thx for that! I think you should remove 
```json
    "extra": {
        "map": [
            [
                "*",
                "Omikron/Factfinder"
            ]
        ]
    }
```
though. This isn't necessary anymore (see also https://github.com/magento/magento2/issues/2433#issuecomment-158078172).

Unless you have deliberately done this, because Omikron expects you to actually change the source code of the [ResultRefiner](https://github.com/FACT-Finder-Web-Components/magento2-module/blob/v0.9-beta.5/Omikron/Factfinder/Helper/ResultRefiner.php#L25-L36). In my opinion it should not be dones this way. Instead, the extension should simply trigger an event for example.

Even if there is no event, you could simply replace the class via your own `di.xml` - which is of course never recommended, but still better than changing the source code of a file directly.

**Comment by a-laurowski** [comment]
@fritzmg 
Hi, I've also remove this extra mapping. It's more than unnecessary, its cause module to be installed in both vendor and app/code directories which I thin, is not an expected result :)

File: composer.json [hunk 1]
```diff
@@ -8,23 +8,15 @@
...
     },
     "type": "magento2-module",
-    "version": "0.1.0",
+    "version": "v0.9-beta.5",
     "license": [
     ],
     "autoload": {
         "files": [
-            "registration.php"
+            "./registration.php"
```
Comment (fritzmg) [inline composer.json]: btw. technically this is not necessary either, is it?

**Comment by fritzmg** [comment]
Btw. what about leaving the `registration.php` within `Omikron/Factfinder` and using
```json
    "autoload": {
        "files": [
            "Omikron/Factfinder/registration.php"
        ],
        …
    }
```
? This way if someone still wants to install this extension manually into `app/code`, they can just extract the `Omikron` folder from the download archive to `app/code`.

**Pull request merged**